### PR TITLE
Timer creation form

### PR DESCRIPTION
**Pull request by hjwk**
Closes BGWE/BoardgameWE#116

### CODE REVIEW

**Comment by hjwk** [comment]
@FServais I took the opportunity to rename some of the routes to lower case.

File: boardgame-ui/src/components/timer/PlayerTimer.vue [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default {
   },
   computed: {
     textColor() {
-      const match = this.player_timer.color.match(/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})$/);
+      const match = this.player_timer.color.match(/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})?$/);
```
Comment (waliens) [inline boardgame-ui/src/components/timer/PlayerTimer.vue]: maybe we should use the transparency in some way if there is one ?

File: boardgame-ui/src/pages/EventCreationPage.vue [hunk 2]
```diff
@@ -42,9 +42,7 @@
         </b-field>
 
         <b-field horizontal>
-          <button type="submit" class="button is-primary">
-            {{$t('event.edition.submit')}}
-          </button>
+          <button type="submit" class="button is-primary">{{$t('event.edition.submit')}}</button>
```
Comment (waliens) [inline boardgame-ui/src/pages/EventCreationPage.vue]: you should use the `<i18n>` tag to inject localized content (see in home page for an exemple).

File: boardgame-ui/src/components/timer/PlayerTimer.vue (hunk 1 above)
Comment (hjwk) [reply boardgame-ui/src/components/timer/PlayerTimer.vue]: why not but maybe in another pull request

File: boardgame-ui/src/pages/EventCreationPage.vue (hunk 2 above)
Comment (hjwk) [reply boardgame-ui/src/pages/EventCreationPage.vue]: According to @elodieburtin you can use the $t('...) syntax just fine. <i18n> is only really needed when you need to inject some html into the translation.

**Pull request merged**